### PR TITLE
FIX some Italian translations: other will come early

### PR DESCRIPTION
**Pull request by DepaMarco**
@7omate I started fixing current Italian translations and hope to finish all strings today or next week.
Is there something that I must change in my work-procedure?

### CODE REVIEW

File: src/pydio/res/i18n/it.po [hunk 1]
```diff
@@ -6,42 +6,43 @@
...
 #, python-format
 msgid "Folder created at %s"
-msgstr "Cartella creata al "
+msgstr "Cartella creata il %s"
 
 #: job/change_processor.py:185
 #, python-format
 msgid "Deleted file %s"
-msgstr "File %s sopresso"
+msgstr "File %s eliminato"
```
Comment (7omate) [inline src/pydio/res/i18n/it.po]: Are you sure elimininato and not not eliminat**a**

File: src/pydio/res/i18n/it.po [hunk 2]
```diff
@@ -6,42 +6,43 @@
...
 
 #: job/change_processor.py:191
 #, python-format
 msgid "Folder %s deleted"
-msgstr "Cartella %s sopressa"
+msgstr "Cartella %s eliminato"
```
Comment (7omate) [inline src/pydio/res/i18n/it.po]: Eliminat**a** ?

File: src/pydio/res/i18n/it.po [hunk 3]
```diff
@@ -56,57 +57,60 @@ msgstr "File %s inviato al server"
...
-"prima di riprovare"
+msgstr ""
+"Impossibile trovare la cartella locale! E' stato rimosso un volume? "
+"Attendere %s secondi prima di riprovare"
 
+# Are you sure that the right word is "volume"? I suggest "unit"
 #: job/continous_merger.py:433
 msgid "Cannot find local folder, did you disconnect a volume?"
-msgstr "Imposibile trovare la cartella locale, avete staccato un volume?"
+msgstr "Impossibile trovare la cartella locale. E' stato rimosso un volume?"
```
Comment (7omate) [inline src/pydio/res/i18n/it.po]: È

**Comment by 7omate** [comment]
That looks great, I added a few notes, if you can delete or respond to the ones you've seen. Thanks a lot for your help.

File: src/pydio/res/i18n/it.po (hunk 1 above)
Comment (DepaMarco) [inline src/pydio/res/i18n/it.po]: Yeah, sure :)

File: src/pydio/res/i18n/it.po (hunk 2 above)
Comment (DepaMarco) [inline src/pydio/res/i18n/it.po]: My fault! :( sorry... the right one is "eliminata" :) I commit the change to this with the other strings

File: src/pydio/res/i18n/it.po (hunk 3 above)
Comment (DepaMarco) [inline src/pydio/res/i18n/it.po]: Yeah, both are correct :) I can change it with the special one, on next commit.

**Pull request merged**